.github/workflows/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: File a bug report
+labels: bug
+
+---
+
+<!-- Thank you for submitting a bug report! All fields are required unless marked optional. -->
+
+## Steps to reproduce
+<!-- Please enable debug logging by running `juju model-config logging-config="<root>=INFO;unit=DEBUG"` (if possible) -->
+1. 
+
+## Expected behavior
+
+
+## Actual behavior
+<!-- If applicable, add screenshots -->
+
+
+## Versions
+
+<!-- Run `lsb_release -sd` -->
+Operating system: 
+
+<!-- Run `juju version` -->
+Juju CLI: 
+
+<!-- Model version from `juju status` -->
+Juju agent: 
+
+<!-- App revision from `juju status` or (advanced) commit hash -->
+Charm revision: 
+
+<!-- Run `lxd version` -->
+LXD: 
```

Comment:
Not required here, maybe `snap revision` instead.

Reply:
That is correct. I will copy the issue template from opensearch